Dockerfile

```
@@ -1,8 +1,23 @@
-FROM cirrusci/flutter:2.8.1
+FROM cimg/android:2021.12.2-node
 
-RUN apt update && apt install -y curl git unzip xz-utils zip libglu1-mesa wget libsqlite3-dev libssl-dev lcov zlib1g-dev build-essential libssl-dev libreadline-dev libyaml-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev g++ gcc autoconf automake bison libc6-dev libffi-dev libgdbm-dev libncurses5-dev libtool libyaml-dev make pkg-config zlib1g-dev libgmp-dev libreadline-dev libssl-dev android-sdk ruby ruby-dev
-RUN mkdir -p Android/sdk
-ENV ANDROID_SDK_ROOT /home/developer/Android/sdk
-RUN mkdir -p .android && touch .android/repositories.cfg
-RUN gem install fastlane -NV 
+USER circleci
 
+RUN sudo apt update
+RUN sudo apt install -y sqlite3 libsqlite3-dev
+
+WORKDIR /home/circleci
+
+# Install brew
+RUN /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+ENV PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH"
+
+# Install fvm
+RUN brew tap leoafarias/fvm
+RUN brew install fvm
```

We could also install FVM via dart global packages. Then we don't need homebrew in our docker container.

```suggestion
RUN dart pub global activate fvm
```

---

`brew install fvm` also includes Dart SDK. If we remove brew for fvm, then we need to use brew for Dart. 

Note: Flutter SDK is not in the image, thus we need to explicit install Dart